config.js

```diff
@@ -0,0 +1,19 @@
+
+module.exports = {
+    extends: ["group:allNonMajor", "config:recommended"],
+    timezone: "Europe/Berlin",
+    dependencyDashboard: true,
+    logFile: "renovate.log",
+    repositories: ['mirek163/api-layer'],
+    baseBranches: ["v2.x.x-renovate","v3.x.x-renovate"],
+    assignees: ["mirek163"],
+    reviewers: ["mirek163"],
+    schedule: ["after 6am every weekday"],
```

Comment:
Which scedule is actually used, this one or that one in the action, or both?

Reply:
Action schedule was used, i deleted this unnecessary schedule inside config of renovate